pages/common/calligraflow.md

```diff
@@ -0,0 +1,16 @@
+# calligraflow
+
+> Calligra's flowchart and diagram application.
+> More information: <https://manned.org/calligraflow>.
+
+- Launch the flowchart and diagram application:
+
+`calligraflow`
+
+- Open a specific file:
+
+`calligraflow {{path/to/file}}`
+
+- Display a help or version:
+
+`calligraflow --{{help|version}}`
```

Comment:
```suggestion
- Display help:

`calligraflow --help`

- Display version:

`calligraflow --version`
```

Since this page doesn't have a lot of commands I think it is better to separate the examples.

(or)

```suggestion
- Display help or version:

`calligraflow --{{help|version}}`
```

Reply:
Done. :)